README.org

```diff
@@ -122,13 +122,16 @@ required in a user's config via =use-package= or =leaf=.
         alwaysTangle = true;
 
         # Optionally provide extra packages not in the configuration file.
-        extraEmacsPackages = epkgs: [
+        # This can also include extra executables to be run by Emacs (linters,
+        # language servers, formatters, etc)
+        extraPackages = epkgs: [
           epkgs.cask
+          pkgs.shellcheck
         ];
```

Comment:
Note that renaming `extraEmacsPackages` to `extraPackages` is a **breaking change**.

Adding comment alone is good and `extraEmacsPackages` may not be a good name with the added comment.  However, I do not think the breaking change is justified.  

WDYT?

Reply:
Ok, I'll leave the README edit, and the option rename is up to the maintainers